engine/src/main/java/org/terasology/utilities/procedural/PerlinNoise.java

```diff
@@ -27,38 +27,50 @@
 public class PerlinNoise extends AbstractNoise implements Noise2D, Noise3D {
 
     private final int[] noisePermutations;
+    private final int permCount;
 
     /**
      * Init. a new generator with a given seed value.
      *
      * @param seed The seed value
      */
     public PerlinNoise(long seed) {
+        this(seed, 1 << 8);
```

Comment:
Is there a specific reason for writing `1 << 8` instead of `256`? Is the grid dimension required to be a power of 2?

Reply:
Ah nothing it think this is copypasta effects?
Anyways fixing this